oggm/utils/_downloads.py

```diff
@@ -526,18 +532,25 @@ def _download_tandem_file_unlocked(zone):
     if os.path.exists(outpath):
         return outpath
 
+    # Grab auth parameters
+    tdmauthfile = os.path.expanduser('~/.tdmdem90.creds')
+    if not os.path.isfile(tdmauthfile):
+        raise DownloadCredentialsMissingException(
+            tdmauthfile + ' does not exist.')
```

Comment:
Add instructions on how to generate this file in the error message.

Reply:
You think what I added is enough? I don't want to write too many instructions into an exception message, but what's there now should be enough to allow figuring things out.